assets/model_monitoring/components/src/data_quality_compute_metrics/compute_data_quality_metrics.py

```diff
@@ -376,6 +386,29 @@ def impute_categorical_with_mode(df: pyspark.sql.DataFrame) -> pyspark.sql.DataF
     return df
 
 
```

Comment:
Could you please add some comments here, for a new mapping added her, please added in the supported_datatypes array.

Reply:
Done in the new PR: https://github.com/Azure/azureml-assets/pull/1533